src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const calculator = (argument) => {
+    if (typeof argument !== 'number') {
```

Comment:
I can suggest to make check in such way, but your variant also fine:
```suggestion
    if (!argument) {
```